corehq/apps/linked_domain/models.py

```diff
@@ -66,6 +68,12 @@ def remote_details(self):
     def is_remote(self):
         return bool(self.remote_base_url) or 'http' in self.linked_domain
 
+    @property
+    def has_full_privilege(self):
```

Comment:
How about calling this `has_full_access`, too?

Reply:
See fbc681ac645d9d1d5a2048db5f7d17aceaf32802